lib/PicoDb/Table.php

```diff
@@ -401,12 +401,20 @@ public function exists()
      * Count
      *
      * @access public
+     * @param string $column
+     * @param bool $distinct
      * @return integer
      */
-    public function count()
+    public function count(string $column = '*', bool $distinct = false)
     {
+        $column = $this->db->escapeIdentifier($column);
```

Comment:
Quoting the column wildcard `*` will result in it being treated as a literal column name. Special handling needs to be implemented for asterisk so that it is passed through unaffected.

Reply:
@joshmcrae that explains the failing unit tests.
1) MysqlTableTest::testCount PicoDb\SQLException: SQL Error: SQLSTATE[42S22]: Column not found: 1054 Unknown column '*' in 'field list'

Would you feel better about doing this logic in db->escapeIdentifier or table->count?

Reply:
Inside `count()`. The other function already has a clear and defined purpose, so passing something which isn't an identifier doesn't sit right with me.